### PR TITLE
Added last edited date to courses API

### PR DESCRIPTION
**Pull request by Nickersoft**
@clintonb @jimabramson FYI: @rlucioni @wedaly

### CODE REVIEW

File: ecommerce/extensions/api/serializers.py [hunk 1]
```diff
@@ -121,8 +125,8 @@ def get_products_url(self, obj):
 
     class Meta(object):
         model = Course
-        fields = ('id', 'url', 'name', 'type', 'products_url',)
-        read_only_fields = ('type',)
+        fields = ('id', 'url', 'name', 'type', 'products_url', 'last_edited')
+        read_only_fields = ('type')
```
Comment (clintonb) [inline ecommerce/extensions/api/serializers.py]: Revert this line.

**Comment by jimabramson** [comment]
tests?  also, does `obj.history.latest()` ever not have a value?

File: ecommerce/extensions/api/serializers.py (hunk 1 above)
Comment (Nickersoft) [inline ecommerce/extensions/api/serializers.py]: @clintonb Should `last_edited` be added to read_only_fields instead? If it is omitted, an error occurs

**Comment by Nickersoft** [comment]
@jimabramson The only time there shouldn't be a value for `obj.history.latest()` is when the course is first created. Then I believe it returns an empty string. Tests were overlooked... I'll append a test for `last_edited` to the current courses API tests.

File: ecommerce/extensions/api/serializers.py (hunk 1 above)
Comment (clintonb) [inline ecommerce/extensions/api/serializers.py]: Revert the deletion of the comma. The `fields` declaration is fine as you have it.

Comment (Nickersoft) [inline ecommerce/extensions/api/serializers.py]: @clintonb Oh, okay. I didn't notice that. Will revert.

**Comment by clintonb** [comment]
@jimabramson @Nickersoft There will always be a last-edited date.

**Comment by Nickersoft** [comment]
@clintonb Created date is included in `last_edited`? I thought there was a previous occasion were the course had to be updated in order for `last_edited` to have a value. It's good to know, nevertheless.

**Comment by Nickersoft** [comment]
@clintonb @jimabramson Added tests

File: ecommerce/extensions/api/serializers.py (hunk 1 above)
Comment (Nickersoft) [inline ecommerce/extensions/api/serializers.py]: Fixed

**Comment by clintonb** [comment]
:+1: (once tests pass)

**Comment by Nickersoft** [comment]
@jimabramson ?

**Comment by rlucioni** [comment]
Looks good, :+1: .

**Comment by Nickersoft** [comment]
@jimabramson Closing due to two :+1:

**Pull request merged**